content/videos/git-github-poets/1-fundamentals/2-branches/index.json

```
@@ -0,0 +1,31 @@
+{
+    "title": "Branches",
+    "description": "In this second video of the Git and GitHub for Poets tutorial series, we cover the concept of branches.",
+    "languages": [],
+    "topics": ["For beginners","Git","GitHub"],
+    "videoId": "oPpnCh7InLY",
+    "timestamps": [
+        { "time": "0:00", "title": "What are branches" },
+        { "time": "2:42", "title": "Branching on GitHub" },
+        { "time": "5:05", "title": "Pull Request and Merging" },
+        { "time": "9:17", "title": "Modify both master and another branch" }
+    ],
+    "groupLinks":[
+        {
+            "title":"References",
+            "links":[
+                {
+                    "title": "GitHub",
+                    "url": "https://github.com",
+                    "description": "GitHub homepage"
+                },
+                {
+                    "title": "GitHub Docs",
+                    "url": "https://docs.github.com/en/get-started/quickstart/hello-world",
+                    "description": "Get started with GitHub"
```

```suggestion
                    "description": "Quickstart tutorial for how to get started with Git and GitHub."
```

These can be full, descriptive sentences if you like!